clients/js/src/index.ts

```diff
@@ -39,3 +39,61 @@ export class CacReader {
         return deepMerge(targetConfig, ...requiredOverrides);
     }
 }
+
+export class ExperimentReader {
+    experiments: ExperimentStore;
+
+    constructor(experiments: Experiments) {
+        this.experiments = new Map(experiments.map(exp => [exp.id, exp]));
+    }
+
+    public getApplicableVariant(data: IObject, toss: number): Array<String> {
```

Comment:
the type `number` allows floating point values too, can we add an assertion that toss is always a positive integer?

Reply:
Can we add the toss isInteger check here?